guides/source/active_model_basics.md

```diff
@@ -449,32 +1314,49 @@ end
 ```
 
 With the `human_attribute_name` method, you can transform attribute names into a
-more human-readable format. The human-readable format is defined in your locale file(s).
-
-* config/locales/app.pt-BR.yml
+more human-readable format. The human-readable format is defined in your locale
+file(s).
 
 ```yaml
+# config/locales/app.pt-BR.yml
 pt-BR:
   activemodel:
     attributes:
       person:
-        name: 'Nome'
+        name: "Nome"
 ```
 
-```ruby
-Person.human_attribute_name('name') # => "Nome"
+```irb
+irb> Person.human_attribute_name("name")
+=> "Name"
+
+irb> I18n.locale = :"pt-BR"
+=> :"pt-BR"
+irb> Person.human_attribute_name("name")
+=> "Nome"
 ```
 
 ### Lint Tests
```

Comment:
This is food for thought, so I'm sharing for discussion in case others think it makes sense: while all other sections more or less follow an alphabetical order (it's not all, some are mixed, but it sort of starts like that), I believe `Lint::Tests` were put last because they are much less relevant and only really necessary to understand if you're not using Model or API yourself, i.e. if you're building some object that needs compliance with Action Pack / View somehow.

Later `SecurePassword` was introduced, and it was put last in the guide, I'm assuming it was just "appended" as new documentation, rather than putting it in any specific order.

I guess what I'm saying is that while it might not make sense to move more sections around, it might perhaps make sense to at least swap these and have `Lint::Tests` be last?

Reply:
I was thinking about the order as well - I added the new sections in alphabetic order but didn't change the order of the existing ones.  

> I guess what I'm saying is that while it might not make sense to move more sections around, it might perhaps make sense to at least swap these and have Lint::Tests be last?

Looking at the index, I do like the thought of "loosely" alphabetizing the sections (in this particular guide) especially since the sections are scoped and not dependant on each other. "Loosely alphabetizing" means we could move Lint::Tests to the last position since they're less relevant.

Right now everything under Dirty looks a little random: 
<img width="307" alt="Screenshot 2024-02-26 at 22 48 42" src="https://github.com/Ridhwana/rails/assets/2786819/1ce52bf2-3146-4a2a-9607-9156f663f77c">

and it may look better like this: 
<img width="247" alt="Screenshot 2024-02-26 at 23 07 24" src="https://github.com/Ridhwana/rails/assets/2786819/e3bcbe56-2692-4b59-b0c9-b6a178297741">

If we don't want to do this though, I think moving `Lint::Tests` to the last position makes sense.

Reply:
I like that :+1: